allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/interceptors/testrun/MoveReportsInterceptor.kt

```diff
@@ -0,0 +1,133 @@
+package com.kaspersky.components.alluresupport.interceptors.testrun
+
+import android.app.Instrumentation
+import androidx.test.uiautomator.UiDevice
+import com.google.common.io.CharStreams
+import com.kaspersky.kaspresso.files.dirs.DirsProvider
+import com.kaspersky.kaspresso.files.resources.ResourcesRootDirsProvider
+import com.kaspersky.kaspresso.interceptors.watcher.testcase.TestRunWatcherInterceptor
+import com.kaspersky.kaspresso.testcases.models.info.TestInfo
+import io.qameta.allure.kotlin.Allure
+import org.json.JSONObject
+import java.io.File
+
+private const val ATTACHMENTS_JSON_FIELD = "attachments"
```

Comment:
nitpick:
why not to simply use private companion object? The separated generated class containing these constants seems redundant.

Reply:
Personally I don't care about generated classes. This way just reads more clearly